### DISCUSSION TITLE
'Hide empty columns' option in kanban

### DISCUSSION THREAD

**Discussion by weitzhandler** [community/community#118825] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Projects

### Body

Currently it's possible to hide individual columns via the 'Hide from view' option in menu.
![image](https://github.com/community/community/assets/2716316/782d1335-3cb6-4a78-8fb9-b4a00762ee10)

My request is to auto hide all columns who have no issues/PRs in them.

I know there's

### Guidelines

- [x] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by erikhuizinga** [comment]
I'd like to suggest a variant, where the empty columns remain visible but collapsed (e.g. narrower), or in a smaller area on the screen or something. This would allow for dragging and dropping to empty columns, and have columns collapse/expand when they become empty/non-empty.
